docs/source/getting_started/content/proc_configuring-the-virtual-machine.adoc

```diff
@@ -0,0 +1,50 @@
+[id="configuring-the-virtual-machine_{context}"]
+= Configuring the virtual machine
+
+The default number of vCPUs and amount of memory available to the {prod} virtual machine can be configured using the `cpus` and `memory` configurable properties, respectively.
```

Comment:
"can be configured" passive voice. 

In this case, I do understand the word "respectively". You're saying that the "cpus" property defines the number of vCPUs and the "memory" property defines the amount of memory available to the vm.